tests/serialized_test_data_generation/configs/c96_6ranks_baroclinic.yml

```diff
@@ -0,0 +1,327 @@
+data_table: default
+diag_table: gs://vcm-fv3config/config/diag_table/baroclinic_test/v1.0/diag_table
+field_table: gs://vcm-fv3config/config/field_table/baroclinic_test/v1.0/field_table
+experiment_name: c96_6ranks_baroclinic
+forcing: gs://vcm-fv3config/data/base_forcing/v1.1/
+orographic_forcing: gs://vcm-fv3config/data/orographic_data/v1.0
+# To use internal test cases, point to an empty ICs directory
+initial_conditions: ""
+namelist:
+  amip_interp_nml:
+    data_set: hurrell
+    date_out_of_range: climo
+    do_sst_pert: false
+    interp_oi_sst: true
+    no_anom_sst: true
+    sst_pert: 0.0
+    sst_pert_type: random
+    use_daily: false
+    use_ncep_ice: true
+    use_ncep_sst: true
+  atmos_model_nml:
+    blocksize: -1
+    chksum_debug: false
+    dycore_only: true
+    fdiag: 1.0
+  cires_ugwp_nml:
+    knob_ugwp_azdir:
+    - 2
+    - 4
+    - 4
+    - 4
+    knob_ugwp_doaxyz: 1
+    knob_ugwp_doheat: 1
+    knob_ugwp_dokdis: 0
+    knob_ugwp_effac:
+    - 1
+    - 1
+    - 1
+    - 1
+    knob_ugwp_ndx4lh: 4
+    knob_ugwp_solver: 2
+    knob_ugwp_source:
+    - 1
+    - 1
+    - 1
+    - 0
+    knob_ugwp_stoch:
+    - 0
+    - 0
+    - 0
+    - 0
+    knob_ugwp_version: 0
+    knob_ugwp_wvspec:
+    - 1
+    - 32
+    - 32
+    - 32
+    launch_level: 55
+  cloud_diagnosis_nml:
+    reiflag: 4
+    reimax: 150.0
+    reimin: 10.0
+    rewmax: 10.0
+    rewmin: 5.0
+  coupler_nml:
+    atmos_nthreads: 1
+    calendar: julian
+    current_date:
+    - 2016
+    - 8
+    - 1
+    - 0
+    - 0
+    - 0
+    days: 0
+    dt_atmos: 225
+    dt_ocean: 225
+    hours: 0
+    memuse_verbose: false
+    minutes: 3
+    months: 0
+    ncores_per_node: 32
+    seconds: 45
+    use_hyper_thread: false
+  diag_manager_nml:
+    max_axes: 240
+    max_files: 106
+    max_num_axis_sets: 100
+    prepend_date: false
+  external_ic_nml:
+    checker_tr: false
+    filtered_terrain: true
+    gfs_dwinds: true
+    levp: 64
+    nt_checker: 0
+  fms_io_nml:
+    checksum_required: false
+    max_files_r: 100
+    max_files_w: 100
+  fms_nml:
+    clock_grain: ROUTINE
+    domains_stack_size: 3000000
+    print_memory_usage: false
+  fv_core_nml:
+    a_imp: 1.0
+    adjust_dry_mass: false
+    beta: 0.0
+    consv_am: false
+    consv_te: 0.0
+    d2_bg: 0.0
+    d2_bg_k1: 0.2
+    d2_bg_k2: 0.1
+    d4_bg: 0.15
+    d_con: 1.0
+    d_ext: 0.0
+    dddmp: 0.5
+    delt_max: 0.002
+    dnats: 1
+    do_sat_adj: false
+    do_vort_damp: true
+    dwind_2d: false
+    external_ic: false
+    fill: true
+    fill_dp: false
```

Comment:
I am confused as to how this is true for the other configurations and doesn't cause validation issues, we didn't port mix_dp -- looks like it only does something when delp < dpmin, so maybe we have just gotten lucky? It seems like a setting that should be false for all configs